proto/pocket/application/event.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package pocket.application;
+
+option go_package = "github.com/pokt-network/poktroll/x/application/types";
+
+import "cosmos_proto/cosmos.proto";
+
+// EventDelegateeChange is an event emitted whenever an application changes its
+// delegatee gateways on chain. This is in response to both a DelegateToGateway
+// and UndelegateFromGateway message.
+message EventDelegateeChange {
+  string app_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application using cosmos' ScalarDescriptor to ensure deterministic encoding
```

Comment:
Let's add `gateway_address` or `delegatee_address` here too.

Reply:
No we only need the app address there is no need for any other fields, adds bloat we dont use

Reply:
I chimed in on the other thread: https://github.com/pokt-network/poktroll/pull/220#discussion_r1417126855